lib/facter/custom_facts/util/fact.rb

```diff
@@ -223,8 +223,7 @@ def find_first_real_value(resolutions)
       end
 
       def log_fact_path(resolve)
-        fact = resolve.fact
-        log.debug("#{resolve.fact_type} fact #{fact.name} got resolved from: #{fact.location}")
+        log.debug("#{resolve.fact_type} fact #{resolve.fact.name} was resolved from: #{resolve.last_evaluated}")
```

Comment:
Does this test both aggregate and non-aggregate custom facts?

Reply:
I believe the tests run here are sufficient? https://github.com/puppetlabs/facter/blob/9a72dd5/acceptance/tests/custom_facts/custom_facts_aggregate_resolutions.rb